### PR TITLE
[Arm64] Correct cross/arm64/tryrun.cmake

### PR DESCRIPTION
**Pull request by sdmaclea**
Updating setting based on CMake answers running on Ubuntu 16.04 arm64

```
grep _EXITCODE ./bin/obj/Linux.arm64.Checked/CMakeCache.txt
```
```
FILE_OPS_CHECK_FERROR_OF_PREVIOUS_CALL_EXITCODE:INTERNAL=1
GETPWUID_R_SETS_ERRNO_EXITCODE:INTERNAL=0
HAS_POSIX_SEMAPHORES_EXITCODE:INTERNAL=0
HAVE_CLOCK_MONOTONIC_COARSE_EXITCODE:INTERNAL=0
HAVE_CLOCK_MONOTONIC_EXITCODE:INTERNAL=0
HAVE_CLOCK_THREAD_CPUTIME_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_ACOS_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_ASIN_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_ATAN2_EXITCODE:INTERNAL=0grep _EXITCODE ./bin/obj/Linux.arm64.Checked/CMakeCache.txt
FILE_OPS_CHECK_FERROR_OF_PREVIOUS_CALL_EXITCODE:INTERNAL=1
GETPWUID_R_SETS_ERRNO_EXITCODE:INTERNAL=0
HAS_POSIX_SEMAPHORES_EXITCODE:INTERNAL=0
HAVE_CLOCK_MONOTONIC_COARSE_EXITCODE:INTERNAL=0
HAVE_CLOCK_MONOTONIC_EXITCODE:INTERNAL=0
HAVE_CLOCK_THREAD_CPUTIME_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_ACOS_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_ASIN_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_ATAN2_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_LOG10_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_LOG_EXITCODE:INTERNAL=0
HAVE_COMPATIBLE_POW_EXITCODE:INTERNAL=0
HAVE_LARGE_SNPRINTF_SUPPORT_EXITCODE:INTERNAL=0
HAVE_MMAP_DEV_ZERO_EXITCODE:INTERNAL=0
HAVE_PROCFS_CTL_EXITCODE:INTERNAL=1
HAVE_PROCFS_MAPS_EXITCODE:INTERNAL=0
HAVE_PROCFS_STATUS_EXITCODE:INTERNAL=0
HAVE_PROCFS_STAT_EXITCODE:INTERNAL=0
HAVE_SCHED_GETCPU_EXITCODE:INTERNAL=0
HAVE_SCHED_GET_PRIORITY_EXITCODE:INTERNAL=0
HAVE_VALID_NEGATIVE_INF_POW_EXITCODE:INTERNAL=0
HAVE_VALID_POSITIVE_INF_POW_EXITCODE:INTERNAL=0
HAVE_WORKING_CLOCK_GETTIME_EXITCODE:INTERNAL=0
HAVE_WORKING_GETTIMEOFDAY_EXITCODE:INTERNAL=0
ONE_SHARED_MAPPING_PER_FILEREGION_PER_PROCESS_EXITCODE:INTERNAL=1
PTHREAD_CREATE_MODIFIES_ERRNO_EXITCODE:INTERNAL=1
REALPATH_SUPPORTS_NONEXISTENT_FILES_EXITCODE:INTERNAL=1
SEM_INIT_MODIFIES_ERRNO_EXITCODE:INTERNAL=1
SSCANF_CANNOT_HANDLE_MISSING_EXPONENT_EXITCODE:INTERNAL=1
SSCANF_SUPPORT_ll_EXITCODE:INTERNAL=0
UNGETC_NOT_RETURN_EOF_EXITCODE:INTERNAL=0
```

### CODE REVIEW

**Comment by sdmaclea** [comment]
@janvorli @jkotas @debayang @dotnet/arm64-contrib PTAL

File: cross/arm64/tryrun.cmake [hunk 1]
```diff
@@ -107,7 +107,7 @@ SET( HAVE_CLOCK_MONOTONIC_COARSE_EXITCODE
      CACHE STRING "Result from TRY_RUN" FORCE)
 
 SET( HAVE_PROCFS_CTL_EXITCODE
```
Comment (janvorli) [inline cross/arm64/tryrun.cmake]: This one is no longer present configure.cmake, so it can be removed.

Comment (sdmaclea) [reply cross/arm64/tryrun.cmake]: Done ++

**Comment by sdmaclea** [comment]
I still see 
```
git grep HAVE_PROCFS_CTL
cross/android/arm/tryrun.cmake:SET( HAVE_PROCFS_CTL_EXITCODE
cross/android/arm64/tryrun.cmake:SET( HAVE_PROCFS_CTL_EXITCODE
cross/arm/tryrun.cmake:SET( HAVE_PROCFS_CTL_EXITCODE
cross/arm64/tryrun.cmake:SET( HAVE_PROCFS_CTL_EXITCODE
cross/armel/tryrun.cmake:SET( HAVE_PROCFS_CTL_EXITCODE
cross/x86/tryrun.cmake:SET( HAVE_PROCFS_CTL_EXITCODE
src/pal/src/config.h.in:#cmakedefine01 HAVE_PROCFS_CTL
src/pal/src/configure.cmake:}" HAVE_PROCFS_CTL)
src/pal/src/debug/debug.cpp:#if HAVE_PROCFS_CTL
src/pal/src/debug/debug.cpp:#elif HAVE_TTRACE // HAVE_PROCFS_CTL
src/pal/src/debug/debug.cpp:#endif  // HAVE_PROCFS_CTL
src/pal/src/debug/debug.cpp:#if HAVE_PROCFS_CTL
src/pal/src/debug/debug.cpp:#endif   // HAVE_PROCFS_CTL
src/pal/src/debug/debug.cpp:#if !HAVE_VM_READ && !HAVE_PROCFS_CTL
```

**Comment by sdmaclea** [comment]
Looks like I need to remove the last commit

**Comment by sdmaclea** [comment]
```
08:15:08 CMake Error: TRY_RUN() invoked in cross-compiling mode, please set the following cache variables appropriately:
08:15:08    HAVE_PROCFS_CTL_EXITCODE
```

**Comment by janvorli** [comment]
@sdmaclea Oh, I am really sorry. I was searching for the HAVE_PROCFS_CTL_EXITCODE instead of the HAVE_PROCFS_CTL.

**Comment by sdmaclea** [comment]
> I was searching for the HAVE_PROCFS_CTL_EXITCODE

Made the same mistake, that's why I didn't catch it earlier

It looks only lightly used in pal debug.cpp, but I'll should just revert the last commit for now.

**Comment by janvorli** [comment]
Thank you!

**Comment by sdmaclea** [comment]
Please merge

**Pull request merged**